### PR TITLE
Fix field typing by removing Field handler parameter

### PR DESCRIPTION
**Pull request by clarkperkins**

### CODE REVIEW

File: lowerpines/endpoints/member.py [hunk 1]
```diff
@@ -7,9 +7,11 @@ class Member(AbstractObject, RetrievableObject):
     member_id = Field(api_name='id')
     user_id = Field()
     nickname = Field()
-    muted = Field()
+    muted = Field(handler=None)
     image_url = Field()
-    autokicked = Field()
+    autokicked = Field(handler=None)
+    app_installed = Field(handler=None)
+    guid = Field()
```
Comment (Bigfootjon) [inline lowerpines/endpoints/member.py]: Thank you for adding these!

File: lowerpines/endpoints/message.py [hunk 2]
```diff
@@ -7,24 +7,24 @@
...
     sender_id = Field()
 
-    def __init__(self, gmi, group_id=None, source_guid=None, text=None, attachments=list()):
+    def __init__(self, gmi, group_id=None, source_guid=None, text=None, attachments=None):
         self.gmi = gmi
         self.group_id = group_id
         self.source_guid = source_guid
         self.text = text
-        self.attachments = attachments
+        self.attachments = attachments or []
```
Comment (Bigfootjon) [inline lowerpines/endpoints/message.py]: Good catch

**Comment by clarkperkins** [comment]
Sure, glad to make those changes!  Looking into it a little more the usage of `handler` does seem fairly isolated to just a couple places, so it should be pretty easy to remove.

**Comment by clarkperkins** [comment]
Sorry I didn't get back sooner - yes I have tested, I've got a groupme bot that uses the library, and everything I use is still working fine with these changes!

**Pull request merged**